mne_bids/write.py

```diff
@@ -873,10 +930,25 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     _participants_json(participants_json_fname, True, verbose)
     _scans_tsv(raw, op.join(kind, bids_fname), scans_fname, overwrite, verbose)
 
-    # TODO: Implement coordystem.json and electrodes.tsv for EEG and  iEEG
+    # TODO: Implement coordystem.json and electrodes.tsv for iEEG
     if kind == 'meg' and not emptyroom:
         _coordsystem_json(raw, unit, orient, manufacturer, coordsystem_fname,
-                          overwrite, verbose)
+                          kind, overwrite, verbose)
+
+    elif kind == 'eeg':
+        # We only write EEG electrodes.tsv and accompanying coordsystem.json
+        # if we have LPA, RPA, and NAS available to rescale to a known
+        # coordinate system frame
+        coords = _extract_landmarks(raw.info['dig'])
+        if set(['RPA', 'NAS', 'LPA']) == set(list(coords.keys())):
+            # Rescale to MNE-Python "head" coord system, which is the
+            # "ElektaNeuromag" system, and equivalent to "CapTrak"
+            pass
```

Comment:
here I basically need to ascertain that the coord frame is 'head'

... if one has a  `DigMontage` object, this is quite easy using the [`transform_to_head`](https://martinos.org/mne/stable/generated/mne.channels.DigMontage.html#mne.channels.DigMontage.transform_to_head) method.

I don't know how to do this with `raw`

Reply:
all this is getting deprecated and removed in https://github.com/mne-tools/mne-python/pull/6639

Maybe you want to give it to mne-python#6639 we can also discuss IRL. I would like to hear your thoughts.

Reply:
we resolved this in ourgoogle hangout session, note for future see: https://github.com/mne-tools/mne-bids/issues/264